### PR TITLE
Fix: Virus should enable playing CrashSiteCleanup in solo

### PR DESCRIPTION
**Pull request by nwai90**
Closes https://github.com/bafolts/terraforming-mars/issues

### CODE REVIEW

File: src/cards/base/Virus.ts [hunk 1]
```diff
@@ -37,6 +37,7 @@ export class Virus extends Card implements IProjectCard {
   }
   public play(player: Player, game: Game): PlayerInput | undefined {
     if (game.getPlayers().length === 1) {
+      new RemoveAnyPlants(player, game, 5).execute();
```
Comment (Lynesth) [inline src/cards/base/Virus.ts:40]: One could argue that you can simply set `this.game.someoneHasRemovedOtherPlayersPlants = true;` directly instead.

**Review by Lynesth — APPROVED** [review]

File: src/cards/base/Virus.ts (hunk 1 above)
Comment (nwai90) [reply src/cards/base/Virus.ts:40]: Very true :D Should I?

Comment (bafolts) [reply src/cards/base/Virus.ts:40]: Yes you should. It is far simpler to set `game.someoneHasremovedOtherPlayersPlants`.

Comment (nwai90) [reply src/cards/base/Virus.ts:40]: Done!

**Pull request merged**